### PR TITLE
FAT2-207 find epao detail

### PR DESCRIPTION
**Pull request by dashton82**
Added to FAT endpoint:

- Postcode
- City/Address4
- DeliveryAreaId

### CODE REVIEW

**Review by scottywakefield — APPROVED** [review]

File: src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs [hunk 1]
```diff
@@ -74,15 +74,24 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
...
+                        var validAudiences = new List<string>();
+
+                        if (UseSandbox)
+                        {
+                            validAudiences.Add(Configuration.SandboxApiAuthentication.Audience);
+                            validAudiences.Add(Configuration.SandboxApiAuthentication.ClientId);
+                        }
+                        else
+                        {
+                            validAudiences.AddRange(Configuration.ApiAuthentication.Audience.Split(","));
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs:86]: What is the split about here - why is this needed now?

Comment (dashton82) [reply src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs:86]: This is required to support Managed Service Identity authentication. The audiences are supplied as the app service names and are comma separated within config.

File: src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs [hunk 2]
```diff
@@ -74,15 +74,24 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                     })
                     .AddJwtBearer(o =>
                     {
+                        var validAudiences = new List<string>();
+
+                        if (UseSandbox)
+                        {
+                            validAudiences.Add(Configuration.SandboxApiAuthentication.Audience);
+                            validAudiences.Add(Configuration.SandboxApiAuthentication.ClientId);
```
Comment (jack-education) [inline src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs:82]:
```suggestion
```
The ClientId isn't needed to be a validAudience

File: src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs [hunk 3]
```diff
@@ -74,15 +74,24 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
...
+                        }
+                        else
+                        {
+                            validAudiences.AddRange(Configuration.ApiAuthentication.Audience.Split(","));
+                            validAudiences.Add(Configuration.ApiAuthentication.ClientId);
```
Comment (jack-education) [inline src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs:87]:
```suggestion
```
The ClientId isn't needed to be a validAudience
